.github/workflows/post_submit_check.yml

```diff
@@ -0,0 +1,16 @@
+name: Post submit tests and validations on main development branch 
+on:
+  push:
+    branches:
+      - 'main'
+  repository_dispatch:
+    types: [post-submit-check]
+jobs:
+  Post-Submit-Check-Main-Branch:
+    runs-on: macos-latest
+    steps:
+      - name: Repo Checkout 
+        uses: actions/checkout@v2
+
+      - name: gRPC Cocoapod Podspec Validation 
+        run: scripts/lint_test_cocoapod_spec.sh
```

Comment:
for my curiosity — how would the result of this validation be surfaced?

Reply:
good question. initially we could use the action tab to manually check the state of this workflow (https://github.com/grpc/grpc-ios/actions/workflows/post_submit_check.yml). But we can add a few monitoring (https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/about-monitoring-and-troubleshooting) to surface this up to main page to look at